CDTDatastoreTests/CDTReplicationTests.m

```diff
@@ -206,6 +206,8 @@ - (void)testURLCredsReplacedWithCookieInterceptorPush
     XCTAssertEqualObjects([push.httpInterceptors[0] class], [CDTSessionCookieInterceptor class]);
 }
 
+// this test can only run on macOS and not iOS because it needs to start a server
+#if TARGET_OS_MAC
```

Comment:
This won't have the desired effect as `TARGET_OS_MAC` is considered a superclass of `TARGET_OS_IPHONE` and `TARGET_OS_IOS`, so will be true on iOS as iOS is considered a variant of OSX (see the comments in `TargetConditionals.h`).  Therefore, I think this should be `#if TARGET_OS_MAC && !TARGET_OS_IPHONE` to only work on Mac and not on iOS devices.